net/nimble/host/include/host/ble_store.h

```diff
@@ -31,6 +31,12 @@ extern "C" {
 #define BLE_STORE_OBJ_TYPE_PEER_SEC     2
 #define BLE_STORE_OBJ_TYPE_CCCD         3
 
+/** Failed to persist record; insufficient storage capacity. */
+#define BLE_STORE_EVENT_OVERFLOW        1
+
+/** About to execute a procedure that may fail due to overflow. */
+#define BLE_STORE_EVENT_OVERFLOW_NEXT   2
```

Comment:
I wasn't sure if this event would be used for other record types as well (not just bonds).  Since the first one (OVERFLOW) is used for multiple record types, I opted to keep both generic.

I do agree BONDING is more clear.  However, my only concern is that we'll have to rename it again (change the API) if we want to use it for other record types in the future.